esx_service/vmdk_ops_test.py

```diff
@@ -380,7 +388,10 @@ class VmdkAttachDetachTestCase(unittest.TestCase):
 
     volNamePre = "vol_UnitTest_Attach"
     vm_name = 'test-vm'
-    max_vol_count = 60
+    if config["run_max_attach"]:
+        max_vol_count = MAX_VOL_COUNT_FOR_ATTACH
```

Comment:
Why MAX_VOL_COUNT_FOR_ATTACH? We should either run_max_attach test or not run at all?